### PR TITLE
show rep tooltip everywhere

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
- Refactored displaying reputation into a component `ReputationView`.
- Added two more places.

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Show relative reputation throughout the app](https://trello.com/c/nk0OCPnd/103-show-relative-reputation-throughout-the-app)

File: src/components/Account/ReputationView.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  reputation: number;
+  totalReputation: number;
+}
+
+export default class ReputationView extends React.Component<IProps, null> {
+  public render() {
+    const { daoName, reputation, totalReputation } = this.props;
+
+    return (
+      <Tooltip overlay={<span>{reputation} {daoName || ''} Reputation in total</span>}>
```
Comment (tibetsprague) [inline src/components/Account/ReputationView.tsx]: the reputation count here should be fixed/rounded to two decimals

Comment (tibetsprague) [reply src/components/Account/ReputationView.tsx]: also should probably use toLocaleString() to look nicer

Comment (dev-matan-tsuberi) [reply src/components/Account/ReputationView.tsx]: https://github.com/daostack/alchemy/pull/71/commits/b0a1d80240a5c5b13a4348d00c396741d535a889

**Pull request merged**